# Goal: triggers the controller.
int32 arm
geometry_msgs/PoseStamped desired_pose

int32 LEFT_ARM = 0
int32 RIGHT_ARM = 1
---
# Result:

---
# Feedback: information that may be used to debug the controller
float64[] joint_position_references
float64[] joint_position_errors
float64[] joint_velocity_references
float64[] joint_velocity_errors
